<ui version="4.0" >
 <class>DlgGeneralBase</class>
 <widget class="QWidget" name="DlgGeneralBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>287</width>
    <height>361</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" >
   <property name="leftMargin" >
    <number>0</number>
   </property>
   <property name="topMargin" >
    <number>0</number>
   </property>
   <property name="rightMargin" >
    <number>0</number>
   </property>
   <property name="bottomMargin" >
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Appearance</string>
     </property>
     <layout class="QHBoxLayout" >
      <property name="spacing" >
       <number>6</number>
      </property>
      <property name="leftMargin" >
       <number>9</number>
      </property>
      <property name="topMargin" >
       <number>9</number>
      </property>
      <property name="rightMargin" >
       <number>9</number>
      </property>
      <property name="bottomMargin" >
       <number>9</number>
      </property>
      <item>
       <layout class="QVBoxLayout" >
        <property name="spacing" >
         <number>6</number>
        </property>
        <property name="leftMargin" >
         <number>0</number>
        </property>
        <property name="topMargin" >
         <number>0</number>
        </property>
        <property name="rightMargin" >
         <number>0</number>
        </property>
        <property name="bottomMargin" >
         <number>0</number>
        </property>
        <item>
         <widget class="QCheckBox" name="kcfg_ShowScrollBars" >
          <property name="text" >
           <string>Show scroll&amp;bars</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="kcfg_SyncThumbnailsViewport" >
          <property name="enabled" >
           <bool>true</bool>
          </property>
          <property name="text" >
           <string>Link the &amp;thumbnails with the page</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="kcfg_ShowOSD" >
          <property name="text" >
           <string>Show &amp;hints and info messages</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" >
        <property name="spacing" >
         <number>6</number>
        </property>
        <property name="leftMargin" >
         <number>0</number>
        </property>
        <property name="topMargin" >
         <number>0</number>
        </property>
        <property name="rightMargin" >
         <number>0</number>
        </property>
        <property name="bottomMargin" >
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="pixmapLabel1_2" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="pixmap" >
           <pixmap/>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Minimum</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>20</width>
            <height>1</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2" >
     <property name="title" >
      <string>Program Features</string>
     </property>
     <layout class="QHBoxLayout" >
      <property name="spacing" >
       <number>6</number>
      </property>
      <property name="leftMargin" >
       <number>9</number>
      </property>
      <property name="topMargin" >
       <number>9</number>
      </property>
      <property name="rightMargin" >
       <number>9</number>
      </property>
      <property name="bottomMargin" >
       <number>9</number>
      </property>
      <item>
       <layout class="QVBoxLayout" >
        <property name="spacing" >
         <number>6</number>
        </property>
        <property name="leftMargin" >
         <number>0</number>
        </property>
        <property name="topMargin" >
         <number>0</number>
        </property>
        <property name="rightMargin" >
         <number>0</number>
        </property>
        <property name="bottomMargin" >
         <number>0</number>
        </property>
        <item>
         <widget class="QCheckBox" name="kcfg_ObeyDRM" >
          <property name="text" >
           <string>&amp;Obey DRM limitations</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="kcfg_WatchFile" >
          <property name="text" >
           <string>&amp;Reload document on file change</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="kcfg_ChooseGenerators" >
          <property name="text" >
           <string>Show backend selection dialog</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" >
        <property name="spacing" >
         <number>6</number>
        </property>
        <property name="leftMargin" >
         <number>0</number>
        </property>
        <property name="topMargin" >
         <number>0</number>
        </property>
        <property name="rightMargin" >
         <number>0</number>
        </property>
        <property name="bottomMargin" >
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="pixmapLabel1_2_2" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="pixmap" >
           <pixmap/>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Minimum</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>20</width>
            <height>1</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3" >
     <property name="title" >
      <string>View Options</string>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <widget class="QCheckBox" name="kcfg_CenterFirstPageInRow" >
        <property name="text" >
         <string>Center &amp;first page</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QLabel" name="columnLabel" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Preferred" hsizetype="Maximum" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text" >
           <string>Overview columns:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="kcfg_ViewColumns" >
          <property name="minimum" >
           <number>3</number>
          </property>
          <property name="maximum" >
           <number>10</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>4</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>DesktopIcon</pixmapfunction>
 <includes>
  <include location="global" >kiconloader.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
